src/Files.Uwp/Filesystem/StorageItems/FtpStorageFile.cs

```diff
@@ -39,7 +38,7 @@ public override string DisplayType
         }
 
         public override DateTimeOffset DateCreated { get; }
-        public override Storage.FileAttributes Attributes { get; } = Storage.FileAttributes.Normal;
+        public override Windows.Storage.FileAttributes Attributes { get; } = Windows.Storage.FileAttributes.Normal;
```

Comment:
Can this be simplified?

Reply:
No, it cannot since we use another namespace which has the same class.
I intentionally don't want it to be simplified because it decreases readability and it **won't work** with Files.Sdk.Storage namespace!!